### DISCUSSION TITLE
업데이트 이후 로그인 안됌

### DISCUSSION THREAD

**Discussion by TTVS6883** [storycraft/node-kakao#351] [Q&A] [answered]
[Error - login] - 로그인을 하는 과정에서 오류가 발생했습니다. 오류 JSON: {
"status": -999,
"message": "최신버전으로 업데이트가 필요합니다."

오늘자 카카오톡 업데이트 이후 로그인이 안되는데,업데이트를 어떻게 하나요 ?
카톡은 모두 업데이트 되었는데도, 저런식으로 로그가 올라오네요

**Comment by storycraft** [comment]
v4미만 버전은 더이상 지원되지 않습니다.
v4로 업데이트해주세요.

**Comment by kiwioverflow** [comment]
사용하시는 node-kakao 버전이 V3 이하일경우, 카카오톡의 구버전(3.1.7 이하) 지원 중단으로 인해 **더이상 로그인이 불가능합니다.** (v3는 3.1.4 사용)

**Comment by jipsoft2** [comment, marked as answer]
client.configProvider.configuration.version = '3.2.7';
client.configProvider.configuration.appVersion = '3.2.7.2777';

로그인 하시기 전에 위 두 줄을 넣어주시면 로그인 됩니다.

**Reply by heeboom** [reply]
정말 감사드립니다.

**Reply by ghost** [reply]
TypeError: Cannot read property 'configuration' of undefined 이런 오류가 뜨는데 어찌 해야할까요...
